src/PrestaShopBundle/Resources/config/services/bundle/bridge.yml

```diff
@@ -0,0 +1,97 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.core.bridge.smarty_bridge:
+    class: PrestaShopBundle\Bridge\Smarty\SmartyBridge
+    arguments:
+      - "@prestashop.adapter.legacy.context"
+      - "@prestashop.core.bridge.breadcrumbs_and_title_configurator"
+      - "@prestashop.core.bridge.footer_configurator"
+      - "@prestashop.core.bridge.header_configurator"
+      - "@prestashop.core.bridge.modal_configurator"
+      - "@prestashop.core.bridge.notification_configurator"
+      - "@prestashop.core.bridge.toolbar_flags_configurator"
```

Comment:
```suggestion
      - !tagged core.bridge.smarty_configurator
```

Reply:
That's what I suggested in my previous comment But it's just FYI it can be done in a later PR